app/logic/solana_requests_logic.rb

```diff
@@ -4,10 +4,14 @@ module SolanaRequestsLogic
   RPC_TIMEOUT = 60 # seconds
 
   # Clusters array, method symbol
-  def solana_client_request(clusters, method, params: [])
+  def solana_client_request(clusters, method, params: nil)
     clusters.each do |cluster_url|
       client = SolanaRpcClient.new(cluster: cluster_url).client
-      result = client.public_send(method, *params).result
+      result =  if params
+                  client.public_send(method, params.first, **params.last).result
+                else
+                  client.public_send(method).result
+                end
```

Comment:
Why is it changed? Did it cause any warnings ?

Reply:
`*params` was causing a warning. Ruby recommendation was to change it to `**params` in order to explicitly convert a Hash into keyword argument.